prepublish.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+
+set -e
+
+RED="\033[0;31m"
+GREEN='\033[0;32m'
+NC='\033[0m' # No color
+
+echo "${GREEN}Pulling latest from GitHub...${NC}"
+git checkout master
+git pull
+
+echo "${GREEN}Cleaning directory and fresh installing...${NC}"
+# TODO: Fix error on yarn build that happens when dist files are removed
+git clean -fdx --exclude="packages/design-system/dist/"
+yarn install
+
+echo "${GREEN}Building files and running tests...${NC}"
+yarn build
+yarn test
+yarn test:e2e --skipBuild
+
+echo "${GREEN}Bumping version and creating tagged release commit...${NC}"
+# Remove package-lock.json to prevent a lerna error
+rm -f "packages/design-system-scripts/package-lock.json"
```

Comment:
Not sure where this `package-lock` is being generated (originally i thought it was from the `npm link`), but it causes an error when lerna tries to make a release commit because `package-lock` is gitignored.